cdhweb/resources/models.py

```diff
@@ -39,20 +35,19 @@ def excerpt(self):
             return self.description
 
 
-
-class UserResource(models.Model):
-    '''Through-model for associating users with resource types and
+class PersonResource(models.Model):
+    '''Through-model for associating people with resource types and
     corresponding URLs for the specified resource type.'''
     resource_type = models.ForeignKey(ResourceType, on_delete=models.CASCADE)
-    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    person = models.ForeignKey("people.Person", on_delete=models.CASCADE)
     url = models.URLField()
```

Comment:
maybe this should be in the people apps instead of resources

Reply:
yep - it got moved in #269